Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python: latest
+
+COPY main.py binanceinfo.py Telegramsender.py .gitignore
+
+CMD ["python", "binanceinfo.py", "Telegramsender.py", ".gitignore",]
```

Comment:
The `CMD` is totaly incorrect. How exactly you plan to launch `.gitignore` files ?